promql/engine.go

```diff
@@ -2607,9 +2607,11 @@ func (ev *evaluator) VectorBinop(op parser.ItemType, lhs, rhs Vector, matching *
 		floatValue, histogramValue, keep, err := vectorElemBinop(op, fl, fr, hl, hr, pos)
 		if err != nil {
 			lastErr = err
+			continue
```

Comment:
It might be possible (in future may be) that `vectorElemBinop` returns an error but we still want the returned values. So can we add another return value to `vectorElemBinop` which will correspond to the value with the `bool` modifier? WDYT?

Reply:
I'd prefer not to add things speculatively - we can always change things later if need be.